spec/mailtrap/contact_lists_api_spec.rb

```
@@ -0,0 +1 @@
+ 
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)